jwst/refpix/refpix_step.py

```diff
@@ -25,6 +25,10 @@ class RefPixStep(Step):
         ovr_corr_mitigation_ftr = float(default=3.0) # Factor to avoid overcorrection of bad reference pixels for IRS2
         preserve_irs2_refpix = boolean(default=False) # Preserve reference pixels in output
         irs2_mean_subtraction = boolean(default=False) # Apply a mean offset subtraction before IRS2 correction
+        refpix_algorithm = option("sirs", default="running_median") # NIR full-frame side pixels algorithm
```

Comment:
This needs to include the default option too.  And maybe call it "median" or "median_filter" instead of "running_median"?

```suggestion
        refpix_algorithm = option("median", "sirs", default="median") # NIR full-frame side pixel algorithm
```